recipe/meta.yaml

```diff
@@ -63,6 +73,7 @@ requirements:
     - python-flatbuffers
     - sympy
     - {{ pin_compatible('numpy') }}
+    - __cuda  # [cuda_compiler_version != "None"]
```

Comment:
Is this actually needed? I believe using the cuda compiler should add a run export for `cudatoolkit` which then appropriately constrains `__cuda` as needed. Normally when someone manually adds the `__cuda` package it's because they're using the CUDA driver API and maybe need a sufficiently new version of the CUDA driver.

Reply:
To be honest, I am not sure about this. I basically copied in a cargo-cult style similar lines in other CUDA-enabled feedstocks, such as:
* https://github.com/conda-forge/jaxlib-feedstock/blob/46a1829880e7012e9a05bf8904ddbbc9e5e51375/recipe/meta.yaml#L67
* https://github.com/conda-forge/pytorch-cpu-feedstock/blob/1a3257e9f1aa9cb3ac8d7347c22493024265f821/recipe/meta.yaml#L108


As I did not know how to quickly check if the `__cuda` run dependency was actually added by `{{ compiler("cuda") }}`. However, it is is completly possible that the explicity dependency is not needed also in those feedstocks. If you are sure that this is not needed we can remove it.

Reply:
I dug into this a bit and it does in fact use the driver library in addition to the runtime library.

That being said, the `compiler("cuda")` will add `cudatoolkit` as a run export: https://github.com/conda-forge/nvcc-feedstock/blob/main/recipe/meta.yaml#L40-L46

And then the `cudatoolkit` package has run constraints for the `__cuda` package: https://github.com/conda-forge/cudatoolkit-feedstock/blob/main/recipe/meta.yaml#L701-L706

So I don't think we should add the `__cuda` package here and we especially shouldn't add it unversioned as we currently do.

Reply:
```suggestion
```

Reply:
Perfect, thanks!